eclair-core/src/test/scala/fr/acinq/eclair/TestUtils.scala

```diff
@@ -28,4 +31,9 @@ object TestUtils {
     .get("buildDirectory") // this is defined if we run from maven
     .getOrElse(new File(sys.props("user.dir"), "target").getAbsolutePath) // otherwise we probably are in intellij, so we build it manually assuming that user.dir == path to the module
 
+  def makeCommitments(channelId: ByteVector32, availableBalanceMsat: Long = 50000000L) =
```

Comment:
This doesn't belong here (this is for methods pertaining to the test environment itself) but probably in `HtlcGenerationSpec` with the reset of the generated content.